lib/Migration/Version1005Date20200312102456.php

```diff
@@ -62,7 +62,7 @@ public function changeSchema(IOutput $output, Closure $schemaClosure, array $opt
 				'notnull' => true,
 				'length' => 256,
 			]);
-			$table->addUniqueIndex(['id'], 'id');
+			$table->addUniqueIndex(['id'], 'e2e_encryption_lock_index');
```

Comment:
`e2e_encryption_lock_index` is 26, adding the oc_ prefix + appending the `_IDX` suffix means it's too long for oracle.
```suggestion
			$table->addUniqueIndex(['id'], 'e2e_unique_lock');
```